### PR TITLE
Updates / Fixes

### PR DESCRIPTION
**Pull request by eracknaphobia**
- Updates for python 3
 - Create resilient session to fix "Max retries exceeded with url" error

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.sling" version="0.0.111" name="Sling" provider-name="d21spike, eracknaphobia">
+<addon id="plugin.video.sling" version="2021.4.2" name="Sling" provider-name="d21spike, eracknaphobia">
```
Comment (eracknaphobia) [inline addon.xml:2]: @d21spike What do you think about switching to a date style versioning?

File: resources/lib/globals.py [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+    import urlparse
+    urlLib = urllib
+    urlParse = urlparse
+else:
+    PY = 3
+    urlLib = urllib.parse
+    urlParse = urlLib
+
+try:
+    xbmc.translatePath = xbmcvfs.translatePath
```
Comment (eracknaphobia) [inline resources/lib/globals.py:21]: This should make translatepath work in both versions of python.

Comment (d21spike) [reply resources/lib/globals.py:21]: I'm ok with the date change ... realistically I was keeping the versioning till we felt it was "Beta"

As far as urllib, I had thought about utilizing [six](https://kodi.wiki/view/Add-on:Kodi_Six) to "normalize" things. We'll still have 2 branches but hopefully this should minimize the discrepancy. 

Edit: Just looked through the changes, you're actually using six like I hoped. Everything else works as far as you know if I fast track this without testing?

Comment (eracknaphobia) [reply resources/lib/globals.py:21]: Yep it should be ready.

**Pull request merged**